client/my-sites/ads/controller.js

```diff
@@ -27,7 +29,9 @@ function _recordPageView( context, analyticsPageTitle ) {
 }
 
 function _getLayoutTitle( context ) {
-	var title = sites.getSelectedSite().jetpack ? 'Ads' : 'WordAds';
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	const title = isJetpackSite( state, siteId ) ? 'Ads' : 'WordAds';
```

Comment:
In any case, I'd suggest skipping this one - it's totally out of this PR's scope.